math/convertall/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 convertall 0.8.0 v
+github.tarball_from releases
+
+python.default_version 39
+
+qt5.min_version     5.4
+
+version             0.8.0
```

Comment:
You can remove this version line; the github.setup line sets the version for you.